xsuite/src/cli/cli.test.ts

```diff
@@ -426,6 +437,9 @@ test.concurrent(
       chalk.cyan(`$ ${binaryPath} .`),
       "",
     ]);
+
+    // Switch back to the default rust version.
+    await c.cmd("install-rust");
```

Comment:
Add a comment here to mention it should be removed eventually

Reply:
Done